core/orchestrator_core.go

```diff
@@ -4391,6 +5697,18 @@ func (o *TridentOrchestrator) ListVolumePublicationsForVolume(
 	return
 }
 
+// listVolumePublicationsForVolume returns a list of VolumePublications for a volume
+// It returns an error if the volume doesn't exist
```

Comment:
The previous public function (```ListVolumePublicationsForVolume()```) returns an error, but this internal function does not.  Is the AI confused between public & internal methods with similar names?